keyboards/hhkb/keymaps/brett/keymap.c

```diff
@@ -0,0 +1,120 @@
+/*  -*-  eval: (turn-on-orgtbl); -*-
+ * Brettm12345 HHKB Layout
+ */
+#include QMK_KEYBOARD_H
```

Comment:
Since you have this in your userspace, and in `brett.h` specifically, you don't actually need this here.

```suggestion
```